source/data/gfdl_archive.rst

```diff
@@ -37,13 +37,13 @@ All users with a GFDL account will have a personal archive directory.
 GFDL Archive Data Structure
 ===========================
 
-The users archive area is located at ``/archive/$USER``.  The ``/archive/$USER``
-is be a symbolic link to ``/arch[0-8bcfgh]/$USER``.  These ``arch[0-8bcfgh]``
-directories are used to better distribute the load on the front-end cache
-drives.
+The users archive area is located at ``/archive/$USER``.  The
+``/archive/$USER`` is be a symbolic link to ``/arch[0-8bcfgh]/$USER``.  These
```

Comment:
"is be a symbolic link"?  (I know it was already there, but if you're changing this anyway...)

Reply:
That's Greek to me -- I don't know how a symbolic link should be discussed. But in English it ought to say "is a symbolic link". But why the hell do you need it at all?